Automattic/jetpack
# AI Assistant: introduce tier plans beta flag

Introduce `enable-tier-plans-ui` beta flag

## Proposed changes:
This PR adds `enable-tier-plans-ui` as a beta extension on the editor to be able to work on tier plans UI without causing regressions.

### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
The flag depends on filter `jetpack_ai_tier_plans_enabled` being enabled or not. Checkout and build plugin (or test on JN). Go to the editor and check on the console:
<img width="514" alt="image" src="https://github.com/Automattic/jetpack/assets/157240/7f4c242f-7282-4adc-b514-4a3b7d223305">

The 'available' value should match the filter enabled/disabled state.
